crypto/stack/stack.c

```diff
@@ -322,17 +327,39 @@ static int internal_find(OPENSSL_STACK *st, const void *data,
     r = ossl_bsearch(&data, st->data, st->num, sizeof(void *), st->comp,
                      ret_val_options);
 
+    if (pnum != NULL) {
+        if (r != NULL) {
+            const void **p = (const void **)r;
+
+            *pnum = 0;
+            while (p < st->data + st->num) {
+                if (st->comp(&data, p) == 0)
```

Comment:
Would it be more readable if this condition were reversed:
``` C
if (st->comp(&data, p) != 0)
    break;
++*pnum;
```

Reply:
yep